hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -332,7 +332,7 @@ public final class HoodieMetadataConfig extends HoodieConfig {
 
   public static final ConfigProperty<Boolean> ENABLE_METADATA_INDEX_PARTITION_STATS = ConfigProperty
       .key(METADATA_PREFIX + ".index.partition.stats.enable")
-      .defaultValue(true)
+      .defaultValue(false)
```

Comment:
With partition stats enabled by default and no columns specified, the partition stats would be calculated for mdt update for every commit for all columns, which can be costly. We will track enabling partition stats by default separately.

Reply:
can someone not enable col stats, but enable only partition stats index. should we auto correct the config value for partition stats index in that case?

Reply:
one can still enable partition stats index keeping col stats disabled. But the config to specify columns for which stats need to be aggregated is common - `hoodie.metadata.index.column.stats.column.list`